src/pipeline.ml

```diff
@@ -327,6 +327,9 @@ let tarides ?app ?notify:channel ?filter ~sched ~staging_auth () =
     ocurrent, "mirage-ci", [
         docker "Dockerfile" ["live", "ocurrent/mirage-ci:live", [`Cimirage "infra_mirage-ci"]]
         ~options:(include_git |> build_kit)
+      ];
+    ocurrent, "solver-service", [
+      docker "Dockerfile" ["live", "ocurrent/solver-service:live", [`Ci4 "infra_solver-service"]];
```

Comment:
Could you extend this to produce builds for arm64 and ppc64?

See the worker at https://github.com/ocurrent/ocurrent-deployer/pull/136/files#diff-5f0eea7390672da3c255b6d2e394932d240c9129a0db3eb95df34ed94a098a43R308